handsontable/src/core.js

```diff
@@ -4496,6 +4496,28 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     callback: () => {
       instance.selectAll();
     },
+  }, {
+    keys: [['Control/Meta', 'Enter']],
+    callback: () => {
+      const selectedRange = instance.getSelectedRange();
+      const {
+        row: highlightRow,
+        col: highlightColumn,
+      } = selectedRange[selectedRange.length - 1].highlight;
+      const valueToPopulate = instance.getDataAtCell(highlightRow, highlightColumn);
+      const cellValues = new Map();
```

Comment:
Is the `Map` interface needed? Maybe simple array structure will be sufficient?

Reply:
There is a reason behind that choice. The Map unifies the keys put into the collection, so there is no coords duplication in the „changes” hooks.

Reply:
Sound reasonable. Maybe extra comment/changing name (or both) to `unifiedCellValues` will protect it against refactor :)? The decision is up to you - PR looks good :white_check_mark: